### PR TITLE
Prevent the external.svg icon to be separated from the link

### LINKED ISSUE #653: Icon for external page can be wrapped to next line
We have an external svg icon added to external links. If the link is at the end of a line, the icon can be wrapped on the next line.

Screenshot 2021-06-20 at 13 41 16

It would be nice if the icon always stayed next to the link, like this:

Screenshot 2021-06-20 at 13 41 37

### PR DESCRIPTION
**Pull request by teoli2003**
Fix #653.

(@schalkneethling : this is my first PR here, so it is probably not good, but let's try 😄  )

### CODE REVIEW

**Comment by schalkneethling** [comment]
> so it is probably not good, but let's try

It’s good 😀

**Review by schalkneethling — APPROVED** [review]

File: sass/atoms/_links.scss [hunk 1]
```diff
@@ -14,6 +14,8 @@ a {
   }
 
   &.external {
+    display: inline-block;
+
```
Comment (schalkneethling) [inline sass/atoms/_links.scss:18]: just a small nit picked up by Prettier. Seems like here are 4 spaces on this line:

```suggestion

```

Comment (schalkneethling) [reply sass/atoms/_links.scss:18]: Going to go ahead and merge the commit suggestions.

File: sass/atoms/_links.scss [hunk 2]
```diff
@@ -14,6 +14,7 @@ a {
   }
 
   &.external {
+    display: inline-block;
```
Comment (schalkneethling) [inline sass/atoms/_links.scss:17]:
```suggestion
    display: inline-block;

```

File: sass/atoms/_links.scss (hunk 1 above)
Comment (teoli2003) [reply sass/atoms/_links.scss]: Thanks!

**Pull request merged**